lib/manageiq/environment.rb

```diff
@@ -54,9 +53,13 @@ def self.while_updating_ui
       end
 
       ui_thread.abort_on_exception = true
+      ui_thread
+    end
 
+    def self.while_updating_ui
```

Comment:
Did you change it to stop using this?

Reply:
@himdel pointed out that `bin/update` still uses this